extensions/container-image/container-image-buildpack/deployment/src/main/java/io/quarkus/container/image/buildpack/deployment/BuildpackProcessor.java

```diff
@@ -42,6 +42,7 @@
 public class BuildpackProcessor {
 
     private static final Logger log = Logger.getLogger(BuildpackProcessor.class);
+    private static final String IN_BUILDPACK_BUILD = "IN_BUILDPACK_BUILD";
```

Comment:
Would it not make more sense to have an `enabled` field in `BuildpackConfig`?

Reply:
I think you are right, we should use real configuration properties.

I am learning towards reusing `quarkus.container-image.build=false` and `quarkus.container-image.push=false` as this does not require us to introduce new properties.

Reply:
Sounds reasonable